src/components/content.js

```diff
@@ -530,7 +526,13 @@ class Content extends React.Component {
 
     // Add drop-specific information to the data.
     data.target = target
-    data.effect = dataTransfer.dropEffect
+
+    // Edge throws "Permission denied" errors when accessing `dropEffect` or `effectAllowed`
```

Comment:
Can you check out the other `COMPAT:` comments in the repo and make this one match those? Thanks